docs/platform/porting/graphics-and-ui.md

```diff
@@ -379,6 +416,47 @@ The output backend interface is mandatory. For more information, see [tdm_backen
+| `hwc_set_commit_handler()`             | Sets a user commit handler. The backend has to call `tdm_hwc_commit_handler` after finishing `hwc_commit`. | Yes  |
+| `hwc_set_property()`                   | Sets the property that has a given property ID by the backend on the hwc object. | Yes  |
+| `hwc_get_property()`                   | Gets the property that has a given property ID by the backend on the hwc object. | Yes  |
+
+The hwc window backend interface is mandatory. For more information, see [tdm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtdm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen).
+
+**Table: HWC window backend interface functions**
+
+| Function                   | Description                              | Mandatory          |
+| -------------------------- | ---------------------------------------- | --------- |
+| `hwc_window_destroy()`              | Destroys `tdm_hwc_window`. The backend module must implement this function. The backend destroys the private window, `tdm_hwc_window`. | Yes |
+| `hwc_window_acquire_buffer_queue()` | Acquires a buffer queue associated with `tdm_hwc_window`. This function can be used when the backend has `TDM_HWC_WIN_CONSTRAINT_BUFFER_QUEUE`. | No  |
+| `hwc_window_release_buffer_queue()` | Releases a buffer queue assoicated with `tdm_hwc_window`. This function can be used when the backend has `TDM_HWC_WIN_CONSTRAINT_BUFFER_QUEUE`. | No  |
+| `hwc_window_set_composition_type()` | Sets the composition type of `tdm_hwc_window`. The backend sets `tdm_hwc_window_composition`.| Yes |
+| `hwc_window_set_buffer_damage()`    | Sets the buffer damage. The backend sets the buffer damage. | Yes |
+| `hwc_window_set_info()`             | Sets the information to `tdm_hwc_window`. The information will be applied when the hwc object is committed. | Yes |
+| `hwc_window_set_buffer()`           | Sets a TDM buffer to `tdm_hwc_window`. A TDM buffer will be applied when the hwc object is committed. | Yes |
+| `hwc_window_set_property()`         | Sets the property that has a given property ID by the backend. | Yes  |
+| `hwc_window_get_property()`         | Gets the property that has a given property ID by the backend. | Yes  |
+| `hwc_window_get_constraints()`      | Gets the constraints of `tdm_hwc_window`. The backend returns `tdm_hwc_window_constraint`. | Yes  |
+| `hwc_window_set_name()`             | Sets the name of `tdm_hwc_window`. The backend can get the name of `tdm_hwc_window`.| Yes  |
+| `hwc_window_set_cursor_image()`     | Sets the cursor memory information associated with `tdm_hwc_window`.	 | Yes  |
 
 The layer backend interface is mandatory. For more information, see [tdm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtdm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen).
```

Comment:
@schanlim 
Thank you for updating the changes.
Line 467:
As per @mijin85cho's previous comment, https://github.com/Samsung/tizen-docs/pull/799#discussion_r428472770, please change "Get" to "Gets".

![799](https://user-images.githubusercontent.com/36875286/84627163-03ff1480-af04-11ea-9271-9174aeec2e87.PNG)

Reply:
I will fix it. thanks. :)